src/main/java/io/nats/client/Options.java

```diff
@@ -1773,4 +1773,12 @@ private void appendOption(CharBuffer builder, String key, char[] value, boolean
             builder.append("\"");
         }
     }
+
+    /**
+     * Set old request style.
+     * @param value true to use the old request style
+     */
+    public void setOldRequestStyle(boolean value) {
+        useOldRequestStyle = value;
+    }

```

Comment:
do you want to allow change style once in use? useOldRequestStyle defaults to false and the builder sets it to true.

Reply:
I have to for Jetstream.  Right now it always needs to be enabled to allow subject overrides, which JS does.